ansible_collections/arista/avd/roles/eos_designs/templates/inband_management/ip-virtual-router-mac-address.j2

```diff
@@ -1,4 +1,6 @@
 {# Set virtual router mac address unless it will be set in network services #}
 {% if not (switch.network_services_l2 is arista.avd.defined(true) and switch.network_services_l3 is arista.avd.defined(true)) %}
+{%     if switch.virtual_router_mac_address is arista.avd.defined(fail_action="error", var_name="virtual_router_mac_address") %}
```

Comment:
Only if this template runs, which means only if `inband_management_subnet` is set.
There is no behavioral change here, we just output a "nicer" error message instead of a generic "Undefined variable" from Jinja.

Reply:
thanks - resolving this!